DXRModules/DeusEx/Classes/DXREventsBase.uc

```diff
@@ -640,14 +640,7 @@ function _AddPawnDeath(ScriptedPawn victim, optional Actor Killer, optional coer
     local string classname;
     local bool dead;
 
-    if (IsHuman(victim.class) && ((damageType == "Stunned") ||
-                        (damageType == "KnockedOut") ||
-                        (damageType == "Poison") ||
-                        (damageType == "PoisonEffect"))){
-        dead = false;
-    } else {
-        dead = true;
-    }
+    dead = !FixScriptedPawn(victim).CanKnockUnconscious(damageType);
```

Comment:
I just realized this won't work in other mods, since we can't modify their ScriptedPawn class (anything in the DXRFixes folder is only for vanilla), I guess this happens before the bStunned variable (or whatever it's called) gets set in the pawn?

worst case this function could be moved into DXREventsBase, maybe with some `if(#defined(___))` stuff, keeping it mostly what the old code was for other mods

Reply:
It doesn't need to be in FixScriptedPawn or anywhere else in particular. All it's doing is checking the class of the pawn. In fact, I originally had it as a static function inside DXRActorsBase. It can easily just be moved back there.

Reply:
Ah, but that was before I was checking mass before a class bLittle property.